app/src/main/java/com/example/nvdovin/weatherapp/presentation/details/DetailActivity.java

```diff
@@ -38,10 +41,11 @@
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
+        requestWindowFeature(Window.FEATURE_NO_TITLE);
```

Comment:
Please crete a theme in styles like FullScreenTheme, and apply this theme from AndroidManifes to activities that must be full screen.